apps/ios/LiveVideo/LiveVideo/Twilio/Chat/ChatMessage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright (C) 2022 Twilio, Inc.
+//
+
+import TwilioConversationsClient
+
+struct ChatMessage: Identifiable {
+    var id: String
```

Comment:
Do these need to be vars? It seems incorrect that you would modify something like the `id` later.

Reply:
This was for UI previews but yeah pushed some improvements.